### ISSUE TITLE
RSS Feed and MailChimp integration

### ISSUE THREAD

**Issue by amita-shukla** [amita-shukla/blog-1#11]
Ensure that the new RSS feed that gets generated should be generated from this site and not the blogger one
- configure Mailchimp that sets the newsletter accordingly
- change the Mailchimp template

**Comment by amita-shukla** [comment]
Fixed the rss feed. The new feed link will be https://amitashukla.in/rss.xml

**Comment by amita-shukla** [comment]
The following further changes need to made to RSS so as to integrate fully into Mailchimp

1. make the description in RSS feed longer (at least the first paragraph)
2. the link of the post in each RSS item is broken
3. provide author in RSS feed, but at the same time remove it from the Mailchimp campaign email template as you're the only author and your name is getting REALLY repetitive
4. add tags to RSS feed. Add those tags as 'categories' item in Mailchimp 
5. add media content to rss feed + mail chimp use the <media:content> tag
6. add social media links if possible, even if not, add them to Mailchimp and test all the links.
7. design the Mailchimp template similar to blog template (font style and color)

**Comment by amita-shukla** [comment]
1. description/excerpt increased to 400 chars
2. fixed link
3. added author in RSS feed
4. added tags. these show as separate `category` for each tag
5. did NOT add media content, as cover image is null for a few posts, tedious null handling here

**Comment by amita-shukla** [comment]
4. added categories in MailChimp
5. I did add media content! fixed a bug in rss-feed related that broke the feed metadata (https://github.com/gatsbyjs/gatsby/issues/16018). The cover image is displayed in the mail now.
6. added social media links, posts can be shared from the inbox as well! Also added profile links in the footer.
7. Changed the background color as the theme color, and changed the font as Open Sans. No other changes made
